src/RestEase/PathSerializationMethod.cs

```diff
@@ -0,0 +1,23 @@
+namespace RestEase
+{
+    /// <summary>
+    /// Type of serialization that should be applied to the path parameter's value
+    /// </summary>
+    public enum PathSerializationMethod
+    {
+        /// <summary>
+        /// Serialized using its .ToString() method
+        /// </summary>
+        ToString,
+
+        /// <summary>
+        /// Serialized using the configured IRequestPathParamSerializer (uses Json.NET by default)
```

Comment:
Nit: Should be `RequestPathParamSerializer` (the doc on `QuerySerializationMethod` is out of date)

Reply:
Does this use `Json.net` by default?

Reply:
Ah that's a leftover from when there was a default JSON.Net serializer, which I later removed. I'm fixing this and the other issues pointed out!